demo_drivers/meshing/mesh_from_gmsh/validate.sh

```diff
@@ -0,0 +1,49 @@
+#! /bin/sh
+
+# Get the OOPMH-LIB root directory from a makefile
+OOMPH_ROOT_DIR=$(make -s --no-print-directory print-top_builddir)
+
+#Set the number of tests to be checked
+NUM_TESTS=3
+
+# Setup validation directory
+#---------------------------
+rm -rf Validation
+mkdir Validation
+
+#######################################################################
+
+# Validation for Airy cantilever
+#-------------------------------
+
+cd Validation
+mkdir RESLT
+
+echo "Running a 3D Gmsh version of Airy cantilever validation "
+../mesh_from_gmsh_solid ../cantilever.msh > OUTPUT
+
+exit
+echo "done"
+echo " " >> validation.log
+echo "3D Gmsh Airy cantilever validation" >> validation.log
+echo "--------------------------" >> validation.log
+echo " " >> validation.log
+echo "Validation directory: " >> validation.log
+echo " " >> validation.log
+echo "  " `pwd` >> validation.log
+echo " " >> validation.log
+cat RESLT/soln0.dat \
```

Comment:
just use one (and only compute one solution). as far as validatino is concerned we're not gaining anything else by doing multiple solves.